src/OmniSharp.Dnx/Utilities/SocketConnection.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Net;
+using System.Net.Sockets;
+using System.Threading;
+using Microsoft.Framework.Logging;
+
+namespace OmniSharp
+{
+    public class SocketConnection
+    {
+        public ILogger Logger { get; private set; }
+        public string ContextFlag { get; private set; }
```

Comment:
Both properties are initialized in constructor only. You can remove the `private set;` here.